app/src/main/java/de/tum/in/tumcampusapp/service/SilenceWorker.kt

```diff
@@ -135,13 +128,23 @@ class SilenceService : JobIntentService() {
 
         private const val RINGER_MODE_SILENT = "0"
 
+        private const val UNIQUE_WORK_NAME = "SilenceWorker_request"
         private fun getWaitDuration(eventDateTime: DateTime): Long {
             val eventTime = eventDateTime.millis
             return Math.min(CHECK_INTERVAL.toLong(), eventTime - System.currentTimeMillis() + CHECK_DELAY)
         }
 
-        @JvmStatic fun enqueueWork(context: Context, work: Intent) {
-            enqueueWork(context, SilenceService::class.java, SILENCE_SERVICE_JOB_ID, work)
+        @JvmStatic fun enqueueWork(context: Context) {
```

Comment:
As above: is there a reason for keeping `@JvmStatic`?
```suggestion
        fun enqueueWork(context: Context) {
```